src/components/StyledButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { View, StyleSheet } from 'react-native';
+import { Button } from 'react-native-elements';
+
+import globalStyles from '../styles/globalStyles';
+
+type StyledButtonProps = {
+  disabled: boolean;
+  text: string;
+  onPress: () => void;
+  extraStyles?: object;
+};
+
+function StyledButton({
+  disabled,
+  onPress,
+  text,
+  extraStyles = {},
+}: StyledButtonProps) {
+  return (
+    <View style={[styles.verticallySpaced, globalStyles.mt20, extraStyles]}>
+      <Button
+        disabledStyle={{
+          opacity: 0.4,
+          backgroundColor: '#EB563B',
+        }}
+        buttonStyle={{
+          backgroundColor: '#EB563B',
+        }}
+        titleStyle={{
```

Comment:
The text size should be 21, with 10 px of padding on top and bottom, and 24 px of padding on left and right